dto/src/main/java/org/jboss/pnc/dto/requests/GroupBuildRequest.java

```diff
@@ -35,6 +36,10 @@
 @JsonDeserialize(builder = GroupBuildRequest.Builder.class)
 public class GroupBuildRequest {
 
+    /**
+     * List of group config revisions to be used instead of the latest revisions of the build configs in the group
```

Comment:
Maybe change the wording a bit. It's not a list of group config revisions but build config revisions associated with the group.